tests/poll.rs

```diff
@@ -9,6 +9,7 @@ use std::thread::{self, sleep};
 use std::time::Duration;
 use std::{fmt, io};
 
+#[macro_use]
```

Comment:
Not needed, we don't use any macros from the `util` module.

Reply:
ok, will remove